pkg/operation/botanist/controlplane/kubeapiserver/deployment/monitoring.go

```diff
@@ -0,0 +1,243 @@
+    expr: histogram_quantile(0.9, sum(rate(` + monitoringMetricRequestDurationSecondsBucket + `{verb="WATCH",resource=~"configmaps|deployments|secrets|daemonsets|services|nodes|pods|namespaces|endpoints|statefulsets|clusterroles|roles"}[5m])) by (le,scope,resource))
+    labels:
+      quantile: "0.9"
+  - record: shoot:apiserver_watch_duration:quantile
+    expr: histogram_quantile(0.2, sum(rate(` + monitoringMetricRequestDurationSecondsBucket + `{verb="WATCH",group=~".+garden.+"}[5m])) by (le,scope,resource))
+    labels:
+      quantile: "0.2"
+  - record: shoot:apiserver_watch_duration:quantile
+    expr: histogram_quantile(0.5, sum(rate(` + monitoringMetricRequestDurationSecondsBucket + `{verb="WATCH",group=~".+garden.+"}[5m])) by (le,scope,resource))
+    labels:
+      quantile: "0.5"
+  - record: shoot:apiserver_watch_duration:quantile
+    expr: histogram_quantile(0.9, sum(rate(` + monitoringMetricRequestDurationSecondsBucket + `{verb="WATCH",group=~".+garden.+"}[5m])) by (le,scope,resource))
+    labels:
+      quantile: "0.9"
+  ### API auditlog ###
+  - alert: KubeApiServerTooManyAuditlogFailures
+    expr: sum(rate (` + monitoringMetricAuditErrorTotal + `{plugin="webhook",job="` + monitoringPrometheusJobNameAPIServer + `"}[5m])) / sum(rate(` + monitoringMetricAuditEventTotal + `{job="` + monitoringPrometheusJobNameAPIServer + `"}[5m])) > bool 0.02 == 1
+    for: 15m
+    labels:
+      service: ` + serviceNameAuditlog + `
+      severity: warning
+      type: seed
+      visibility: operator
+    annotations:
+      description: 'The API servers cumulative failure rate in logging audit events is greater than 2%.'
+      summary: 'The kubernetes API server has too many failed attempts to log audit events'
+  ### API latency ###
+  - record: apiserver_latency_seconds:quantile
+    expr: histogram_quantile(0.99, rate(` + monitoringMetricRequestDurationSecondsBucket + `[5m]))
+    labels:
+      quantile: "0.99"
+  - record: apiserver_latency:quantile
+    expr: histogram_quantile(0.9, rate(` + monitoringMetricRequestDurationSecondsBucket + `[5m]))
+    labels:
+      quantile: "0.9"
+  - record: apiserver_latency_seconds:quantile
+    expr: histogram_quantile(0.5, rate(` + monitoringMetricRequestDurationSecondsBucket + `[5m]))
+    labels:
+      quantile: "0.5"
+
+  - record: shoot:kube_apiserver:sum_by_pod
+    expr: sum(up{job="` + monitoringPrometheusJobNameAPIServer + `"}) by (pod)`
+)
+
+var (
+	monitoringAllowedMetrics = []string{
+		monitoringMetricAuditErrorTotal,
+		monitoringMetricAuditEventTotal,
+		monitoringMetricCurrentInflightRequests,
+		monitoringMetricCurrentInqueueRequests,
+		monitoringMetricDroppedRequestsTotal,
+		monitoringMetricRegisteredWatchers,
+		monitoringMetricRequestCount,
+		monitoringMetricRequestDurationSecondsBucket,
+		monitoringMetricRequestTerminationTotal,
+		monitoringMetricRequestTotal,
+		monitoringMetricEtcdObjectCounts,
+		monitoringMetricProcessMaxFds,
+		monitoringMetricProcessOpenFds,
+	}
```

Comment:
There seem to be way more, see https://github.com/gardener/gardener/blob/30f89e1b808262397a278a089f9a849317ebce2a/charts/seed-monitoring/charts/core/charts/prometheus/values.yaml#L71-L101 ?